NEWS.md

```diff
@@ -363,6 +363,9 @@ Deprecated or removed
   * The `cholfact`/`cholfact!` methods that accepted an `uplo` symbol have been deprecated
     in favor of using `Hermitian` (or `Symmetric`) views ([#22187], [#22188]).
 
+  * The `thin` keyword argument for orthogonal decomposition methods has
+    been deprecated in favor of `square` ([#24279]).
```

Comment:
which has the opposite meaning: `thin=true` if and only if `square=false`.

Reply:
Good call! Revised accordingly. Thanks!